rules/integrations/aws/exfiltration_rds_using_snapshot.toml

```diff
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2022/02/03"
+maturity = "production"
+updated_date = "2022/02/03"
+integration = "aws"
+
+[rule]
+author = ["Elastic", "Stijn Holzhauer"]
```

Comment:
If this was solely your work without Elastic collaboration, you can put yourself as a first author and remove Elastic